src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -1412,6 +1412,8 @@ UnitsStyle SetUnitsStyleFromString(std::string const &unitStringIn)
         unitsStyleReturn = UnitsStyle::JtoGJ;
     } else if (UtilityRoutines::SameString(unitStringIn, "INCHPOUND")) {
         unitsStyleReturn = UnitsStyle::InchPound;
+    } else if (UtilityRoutines::SameString(unitStringIn, "INCHPOUNDEXCEPTELECTRICITY")) {
+        unitsStyleReturn = UnitsStyle::InchPoundExceptElectricity;
```

Comment:
@mjwitte Some general answers for now: 

Yes, the original InchPound option does have some capability to do something different (e.g. kW) for electricity---this is based on the string sent in to convert in a couple of special convert functions. However, there are a couple of limitations in the original code: 1). To do this, the code needs to call a special (or two) function call rather than plain conversion; 2). and this is not applied consistently across the report, e.g. in a few large reports such as the end use reports.

I think some of the "unconverted" ones might be in pre-defined tables (see a note in the past paragraph of the top comment) (or this could be some missing ones that could be specially treated in `InchPoundExceptElectrcity` option. For pre-defined tables, it is a little bit (or actually quite difficult) to do something specially for a particular unit in the tabular output section, except that the original `InchPound` way (string based identification for some kW) can be applied (and was already applied in the original code) to do the conversion for electricity. The `InchPoundExceptElectricity` option would just inherit the same treatments as the plain `InchPound` option. Also, for predefined table (or wherever this is applicable), the conversion were be 100 percent correct since it is based on search for a couple of special substrings (like `elec` or `light` etc).

During the development, I realized this issue later in the development process after some more understanding on what the code does. A few thoughts were given to this and it could make the implementation slightly different---for example, this mechanism maybe applied to a few deferent places---but still not 100 percent effective. On the other hand, the InchPoundExceptElectricty option does explicitly do the electricty kW conversion in some tables (e.g. notable ones might be end use, end use sub, and source use tables etc).